src/Model/RedisAdapter.php

```diff
@@ -12,33 +12,25 @@
 
 namespace App\Model;
 
+use App\Entity\User;
 use Pagerfanta\Adapter\AdapterInterface;
 
 /**
  * @author Jordi Boggiano <j.boggiano@seld.be>
  */
 class RedisAdapter implements AdapterInterface
```

Comment:
Let me know if you miss the generic in here ... ;)

Reply:
IIRC, pagerfanta added generics on the Pagerfanta class, but not yet on the AdapterInterface

Reply:
https://github.com/BabDev/Pagerfanta/commit/7fb9610d0a81f2c0b14abddd4f0bc82054da79cc added them on the adapter interface, I just need to remember to release it when I get home tonight.

Reply:
oh great. This commit was not there last time I checked (and doing something like that was on my TODO-list)

Reply:
Implemented this & a couple more related fix I spotted here in https://github.com/composer/packagist/commit/951d9e97ccdf9360c2b0e3bd77f2354937f601b6